sdk/servicebus/service-bus/rollup.base.config.js

```diff
@@ -60,7 +60,7 @@ export function nodeConfig({ test = false, production = false } = {}) {
 
   if (test) {
     // entry point is every test file
```

Comment:
Is the comment still correct?

Reply:
Yes, based on description at https://github.com/rollup/rollup-plugin-multi-entry this seems applicable